cosmic_ray/operators/unary_operator_replacement.py

```diff
@@ -3,19 +3,19 @@
 
 import ast
 
-from .operator import Operator
 from ..util import build_mutations
+from .operator import ReplacementOperator
+
 
 # None indicates we want to delete the operator
-OPERATORS = (ast.UAdd, ast.USub, ast.Invert, ast.Not, None)
+_AST_OPERATORS = (ast.UAdd, ast.USub, ast.Invert, ast.Not)
 
 
 def _to_ops(from_op):
-    """
-        The sequence of operators which `from_op` could be mutated to.
+    """The sequence of operators which `from_op` could be mutated to.
     """
 
-    for to_op in OPERATORS:
+    for to_op in _AST_OPERATORS + (None,):
         if to_op and from_op is ast.Not:
```

Comment:
If this means:

    if (to_op is not None) and (from_op is ast.Not):

I think it would be clearer with the explicit `is` operator and parentheses.  If this isn't what was intended, then either it's wrong or unclear, so needs changing anyway ;-)